gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,504 @@
+.. license-header
+  SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+  SPDX-License-Identifier: Apache-2.0
+
+############################################################
+Install the NVIDIA GPU DRA Driver and Configure IMEX Support
+############################################################
+
+THe NVIDIA GPU DRA Driver is an additional component you can install after the GPU Operator that enables you to use the Kubernetes DRA feature to define IMEX channel resources that are managed by Kubernetes.
+This page details more information about the GPU DRA Driver, including how to install it and examples of deploying workloads using IMEX channels.
+
+*******************************
+About the NVIDIA GPU DRA Driver
+*******************************
+
+The NVIDIA GPU DRA Driver leverages the Kubernetes Dynamic Resource Allocation (DRA) API to support NVIDIA IMEX channels available in GH200 and GB200 GPUs.
+The NVIDIA GPU DRA Driver creates and manages IMEX channels through the creation of a ComputeDomain custom resource. You use this custom resource to define your resource templates, and then reference the templates within your workload specs. The GPU DRA Driver will handle creating the cooresponding IMEX channels for you. 
+
+An IMEX channel is a construct that allows a set of GPUs to directly read and write each other's memory over a high-bandwidth NVLink. 
+The NVLink connection may either be directly between GPUs on the same node or between GPUs on separate nodes connected by an NVSwitch. 
+Once an IMEX channel has been established for a set of GPUs, they are free to read and write each other's memory via extensions to the CUDA memory call APIs.

```

Comment:
My hunch is that we should not go into this detail here but just link to reference documentation.

Reply:
Reference docs are here, for example: https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__MALLOC__ASYNC.html#group__CUDA__MALLOC__ASYNC_1g8aa4c143dbc20293659cd883232b95f2

There is the beautiful statement of

> When exporter and importer CUDA processes have been granted access to the same IMEX channel, they can securely share memory.

And that's already a better description because of the emphasis on _securely_.

Maybe we can borrow that statement, and otherwise just link to refdocs.